examples/run/run.cpp

```diff
@@ -634,20 +634,20 @@ class LlamaData {
         return path.substr(pos + 1);
     }
 
-    int remove_proto(std::string & model_) {
-        const std::string::size_type pos = model_.find("://");
+    int rm_substring(std::string & model_, const std::string & substring) {
```

Comment:
the function name is a quite misleading because I thought that it's `replace(substr, "")` at first. Probably `rm_until_substring` make it more clear

Reply:
Renamed, please re-review @ngxson